pages/mobile/base.py

```diff
@@ -33,16 +41,42 @@ def menu_items(self):
     def is_mobile_view_displayed(self):
         return self.selenium.find_element(*self._mobile_header_locator).is_displayed()
 
+    @property
+    def is_user_logged_in(self):
+        return self.is_element_present(*self._log_out_menu_locator)
+
     def click_menu(self):
         self.selenium.find_element(*self._menu_button_locator).click()
 
     def close_menu(self):
         assert self.is_menu_exposed, 'Menu is already closed'
         self.click_menu()
+        WebDriverWait(self.selenium, self.timeout).until(
+            lambda s: not self.is_element_visible(*self._search_bar_locator))
```

Comment:
I was expecting this to wait for the menu to be shown, rather than the search bar to be visible. If the latter is enough then perhaps we don't even need `is_menu_exposed`?